industries-cares.json

```diff
@@ -27,59 +27,59 @@
                         "link": {
-                            "label": "Read More",
-                            "url": "#"
+                            "label": "See Pics",
+                            "url": "https://drive.google.com/drive/folders/1a3vNSsQZQyzIKmVWlQx3Taccf81ofef3"
                         }
                     },
                     {
                         "imgSrc": "/assets/Samsung_Smart_Class_picture.jpeg",
-                        "title": "Computer Classes",
+                        "title": "Computer Lab Setup",
                         "description":
-                            "Lorem Ipsum. Lorem ipsum dolor sit amet. Lorem Ipsum. Lorem ipsum dolor sit amet. Lorem Ipsum. Lorem ipsum dolor sit amet. Lorem Ipsum",
+                            "We set up 2x24 switches to accommodate up to 48 desktops and provide 2x the internet connection!",
                         "link": {
-                            "label": "Sign Up",
-                            "url": "#"
+                            "label": "Read More",
+                            "url": "https://org62.lightning.force.com/lightning/r/0D50M00004See6ESAR/view"
                         }
                     },
                     {
                         "imgSrc": "/assets/hands_in.ts_177387006.jpg",
-                        "title": "Donation Drive",
+                        "title": "Computer Classes",
                         "description":
-                            "Lorem Ipsum. Lorem ipsum dolor sit amet. Lorem Ipsum. Lorem ipsum dolor sit amet. Lorem Ipsum. Lorem ipsum dolor sit amet. Lorem Ipsum",
+                            "Weekly programming basics classes to ~60 students from 9th grade. We will end the course with a hackathon!",
                         "link": {
                             "label": "Read More",
-                            "url": "#"
+                            "url": "https://org62.lightning.force.com/lightning/r/0D50M00004XIeBeSAL/view"
                         }
                     }
                 ],
                 "action": {
-                    "description": "Want to see more pictures of these events?",
-                    "label": "View on Drive",
-                    "url": "#"
+                    "description": "Interested in donating to Jyothi Vidyalaya",
+                    "label": "Click Here to Donate",
+                    "url": "https://org62.lightning.force.com/lightning/r/0D50M00004XEkccSAD/view"
                 }
             },
             {
                 "title": "What are we looking at ahead?",
                 "activities": [
                     {
                         "imgSrc": "/assets/00-e1498446216258.jpg",
-                        "title": "Menstrual Hygiene for Girls",
+                        "title": "Menstrual Care for Girls",
```

Comment:
I think we decided to remove "Girls"

Reply:
Hey, I'll fix this in next iteration. Pls approve the current version.